2024-06-22 19:15:53,350 INFO    StreamThr :4080 [internal.py:wandb_internal():85] W&B internal server running at pid: 4080, started at: 2024-06-22 19:15:53.349302
2024-06-22 19:15:53,351 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status
2024-06-22 19:15:53,351 INFO    WriterThread:4080 [datastore.py:open_for_write():87] open: /Users/zhemingan/anaconda3/lib/python3.11/site-packages/signaturesnet/wandb/run-20240622_191553-6lfnq7od/run-6lfnq7od.wandb
2024-06-22 19:15:53,353 DEBUG   SenderThread:4080 [sender.py:send():378] send: header
2024-06-22 19:15:53,353 DEBUG   SenderThread:4080 [sender.py:send():378] send: run
2024-06-22 19:15:53,506 ERROR   SenderThread:4080 [internal_api.py:execute():344] 404 response executing GraphQL.
2024-06-22 19:15:53,506 ERROR   SenderThread:4080 [internal_api.py:execute():345] {"errors":[{"message":"project not found","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-06-22 19:15:53,506 ERROR   SenderThread:4080 [sender.py:send_run():970] [Errno 32] Broken pipe
Traceback (most recent call last):
  File "/Users/zhemingan/anaconda3/lib/python3.11/site-packages/wandb/sdk/internal/internal_api.py", line 340, in execute
    return self.client.execute(*args, **kwargs)  # type: ignore
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/zhemingan/anaconda3/lib/python3.11/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/zhemingan/anaconda3/lib/python3.11/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/zhemingan/anaconda3/lib/python3.11/site-packages/wandb/sdk/lib/gql_request.py", line 59, in execute
    request.raise_for_status()
  File "/Users/zhemingan/anaconda3/lib/python3.11/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://api.wandb.ai/graphql

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/zhemingan/anaconda3/lib/python3.11/site-packages/wandb/apis/normalize.py", line 41, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/zhemingan/anaconda3/lib/python3.11/site-packages/wandb/sdk/internal/internal_api.py", line 2188, in upsert_run
    response = self.gql(
               ^^^^^^^^^
  File "/Users/zhemingan/anaconda3/lib/python3.11/site-packages/wandb/sdk/internal/internal_api.py", line 312, in gql
    ret = self._retry_gql(
          ^^^^^^^^^^^^^^^^
  File "/Users/zhemingan/anaconda3/lib/python3.11/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/zhemingan/anaconda3/lib/python3.11/site-packages/wandb/sdk/internal/internal_api.py", line 347, in execute
    wandb.termerror(f"Error while calling W&B API: {error} ({response})")
  File "/Users/zhemingan/anaconda3/lib/python3.11/site-packages/wandb/errors/term.py", line 64, in termerror
    _log(
  File "/Users/zhemingan/anaconda3/lib/python3.11/site-packages/wandb/errors/term.py", line 103, in _log
    click.echo(line, file=sys.stderr, nl=newline)
  File "/Users/zhemingan/anaconda3/lib/python3.11/site-packages/click/utils.py", line 298, in echo
    file.write(out)  # type: ignore
    ^^^^^^^^^^^^^^^
BrokenPipeError: [Errno 32] Broken pipe

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/zhemingan/anaconda3/lib/python3.11/site-packages/wandb/sdk/internal/sender.py", line 968, in send_run
    server_run = self._init_run(run, config_value_dict)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/zhemingan/anaconda3/lib/python3.11/site-packages/wandb/sdk/internal/sender.py", line 1013, in _init_run
    server_run, inserted, server_messages = self._api.upsert_run(
                                            ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/zhemingan/anaconda3/lib/python3.11/site-packages/wandb/apis/normalize.py", line 87, in wrapper
    raise CommError(message, err).with_traceback(sys.exc_info()[2])
  File "/Users/zhemingan/anaconda3/lib/python3.11/site-packages/wandb/apis/normalize.py", line 41, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/zhemingan/anaconda3/lib/python3.11/site-packages/wandb/sdk/internal/internal_api.py", line 2188, in upsert_run
    response = self.gql(
               ^^^^^^^^^
  File "/Users/zhemingan/anaconda3/lib/python3.11/site-packages/wandb/sdk/internal/internal_api.py", line 312, in gql
    ret = self._retry_gql(
          ^^^^^^^^^^^^^^^^
  File "/Users/zhemingan/anaconda3/lib/python3.11/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/zhemingan/anaconda3/lib/python3.11/site-packages/wandb/sdk/internal/internal_api.py", line 347, in execute
    wandb.termerror(f"Error while calling W&B API: {error} ({response})")
  File "/Users/zhemingan/anaconda3/lib/python3.11/site-packages/wandb/errors/term.py", line 64, in termerror
    _log(
  File "/Users/zhemingan/anaconda3/lib/python3.11/site-packages/wandb/errors/term.py", line 103, in _log
    click.echo(line, file=sys.stderr, nl=newline)
  File "/Users/zhemingan/anaconda3/lib/python3.11/site-packages/click/utils.py", line 298, in echo
    file.write(out)  # type: ignore
    ^^^^^^^^^^^^^^^
wandb.errors.CommError: [Errno 32] Broken pipe
2024-06-22 19:15:55,901 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:15:58,371 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: keepalive
2024-06-22 19:15:58,526 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:16:00,922 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:16:03,398 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: keepalive
2024-06-22 19:16:03,542 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:16:05,941 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:16:08,422 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: keepalive
2024-06-22 19:16:08,558 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:16:10,960 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:16:13,441 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: keepalive
2024-06-22 19:16:13,579 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:16:15,971 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:16:18,461 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: keepalive
2024-06-22 19:16:18,590 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:16:20,992 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:16:23,481 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: keepalive
2024-06-22 19:16:23,612 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:16:26,004 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:16:28,517 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: keepalive
2024-06-22 19:16:28,636 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:16:31,019 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:16:33,547 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: keepalive
2024-06-22 19:16:33,645 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:16:36,036 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:16:38,559 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: keepalive
2024-06-22 19:16:38,657 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:16:41,046 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:16:43,580 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: keepalive
2024-06-22 19:16:43,678 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:16:46,058 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:16:48,606 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: keepalive
2024-06-22 19:16:48,694 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:16:51,077 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:16:53,628 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: keepalive
2024-06-22 19:16:53,716 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:16:56,094 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:16:58,657 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: keepalive
2024-06-22 19:16:58,729 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:17:01,116 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:17:03,747 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:17:06,137 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:17:08,764 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:17:11,152 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:17:13,776 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:17:16,167 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:17:18,784 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:17:21,184 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:17:23,799 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:17:26,202 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:17:28,814 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:17:31,222 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:17:33,829 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:17:36,241 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:17:38,845 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:17:41,255 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:17:43,864 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:17:46,274 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:17:48,885 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:17:51,293 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:17:53,902 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:17:56,315 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:17:58,923 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:18:01,330 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:18:03,943 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:18:06,343 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:18:08,965 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:18:11,357 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:18:13,981 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:18:16,367 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:18:19,003 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:18:21,382 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:18:24,023 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:18:26,396 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:18:29,042 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:18:31,410 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:18:34,060 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:18:36,423 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:18:38,921 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: shutdown
2024-06-22 19:18:38,922 INFO    HandlerThread:4080 [handler.py:finish():882] shutting down handler
2024-06-22 19:18:39,077 DEBUG   HandlerThread:4080 [handler.py:handle_request():158] handle_request: status_report
2024-06-22 19:18:39,433 INFO    SenderThread:4080 [sender.py:finish():1545] shutting down sender
2024-06-22 19:18:39,435 INFO    WriterThread:4080 [datastore.py:close():296] close: /Users/zhemingan/anaconda3/lib/python3.11/site-packages/signaturesnet/wandb/run-20240622_191044-02la6n23/run-02la6n23.wandb
